docs/guides/math-branching-dag-builder.md

```diff
@@ -71,7 +71,7 @@ This DAG uses tools from the `xyz.taluslabs.math.i64` namespace. Based on the st
     *   Output: If successful (`ok` variant), outputs an `i64` named `result`.
 *   `xyz.taluslabs.math.i64.cmp@1`:
     *   Input: Takes two `i64` numbers, typically named `a` and `b`.
-    *   Output: Based on comparing `a` to `b`, it outputs the value of `a` under one of three variants: `lt` (less than), `gt` (greater than), or `eq` (equal to). The output port name in each variant is `a`.
+    *   Output: Based on comparing `a` to `b`, it outputs the value of `a` and `b` under one of three variants: `Lt` (less than), `Gt` (greater than), or `Eq` (equal to).
```

Comment:
the casing of the output variants was correct before

Reply:
reverted